xml/obs_multibuild.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter id="cha.obs.multibuild">
+ <title>Multibuild</title>
+ <section>
+  <title>Overview</title>
+  <para>
+   Multibuild is used for sources that can build multiple packages. For that to
+   be accomplished you must provide a file with a list of package names.
+   For each of these listed packages a build instruction, e.g., a spec file, is
```

Comment:
This comma is not needed: For each of these listed packages a build instruction, e.g.**`,`** a spec file, is